### PR TITLE
Readme Portuguese-Brazil

### PR DESCRIPTION
**Pull request by latreta**
* Created a portuguese version of the file.

### CODE REVIEW

File: README.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,529 @@
+# scrcpy (v1.12.1)
+
+Esta aplicação fornece visualização e controle de dispositivos Androids conectados via
```
Comment (LuccaPrado) [inline README.pt-br.md:3]: Aqui, o correto não seria android no singular? Veja, da forma como foi colocado, é equivalente a:
"controle de dispositivos com sistema operacional  Androids conectados"

Já se estivesse no singular
"controle de dispositivos com sistema operacional Android conectados"

File: README.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+[AUR]: https://wiki.archlinux.org/index.php/Arch_User_Repository
+[aur-link]: https://aur.archlinux.org/packages/scrcpy/
+
+Para Gentoo, uma [Ebuild] está disponível: [`scrcpy/`][ebuild-link].
+
+[Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
+[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
+
+
+Você também pode [buildar a aplicação manualmente][BUILD] (não se preocupe, não é tão difícil).
```
Comment (LuccaPrado) [inline README.pt-br.md:64]: Aqui não seria melhor usar o termo compilar? 
Pois por ser uma documentação, creio que não é tão interessante ser informal

File: README.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+
+
+### Windows
+
+Para Windows, para simplicidade, um arquivo pré-buildado com todas as dependências
```
Comment (LuccaPrado) [inline README.pt-br.md:69]: se seguir minha recomendação sobre o termo "buildar", aqui deverá entrar o "pré-compilado"

File: README.pt-br.md [hunk 4]
```diff
@@ -0,0 +1,529 @@
...
+brew install scrcpy
+```
+
+Você precisa do `adb`, acessível através do seu `PATH`. Se você ainda não o tem:
+
+```bash
+brew cask install android-platform-tools
+```
+
+Você também pode [buildar a aplicação manualmente][BUILD].
```
Comment (LuccaPrado) [inline README.pt-br.md:114]: Caso minha sugestão seja acatada, aqui seria "compilar"

File: README.pt-br.md (hunk 1 above)
Comment (latreta) [reply README.pt-br.md]: Concordo, corrigido.

File: README.pt-br.md (hunk 2 above)
Comment (latreta) [reply README.pt-br.md]: Concordo plenamente, já fiz a mudança ao longo do arquivo.

**Comment by latreta** [comment]
Fixed the gramatical error and changed to better word to describe it according to the reviews.

**Comment by rom1v** [comment]
Thank you for the translation. :+1:

I squashed the changes into one commit (IMO there is no need to keep them split into several commits): [`pr1313`](https://github.com/Genymobile/scrcpy/commits/pr1313).

I don't know the concrete differences between "Brazilian Portuguese" and "Portuguese", but would it be an error to just name the file `README.pt`? (Like for French there is `fr_FR`, `fr_BE` and others, but in practice for a README we don't care.)

**Comment by LuccaPrado** [comment]
Sorry for commenting on closed thread, but, @rom1v , yes. It would be an error to rename it as REDME.pt as there is Pt-Pt and Pt-Br.

**Comment by rom1v** [comment]
OK, but are these languages very different? My argument was that there are fr_FR, fr_BE, fr_CA (for French) or even en_US and en_GB for English, but I would definitely not crease a specific README file for en_GB.

I guess this is different for pt_PT and pt_BR?

**Comment by rom1v** [comment]
Merged into `master`: 24c803924449b1f72801e982ba35dfb80d54116d.

**Pull request closed without merging**